lib/mjsonwp/routes.js

```diff
@@ -369,6 +369,9 @@ const METHOD_MAP = {
   '/wd/hub/session/:sessionId/appium/device/current_package': {
     GET: {command: 'getCurrentPackage'}
   },
+  '/wd/hub/session/:sessionId/appium/device/broadcast_intent': {
```

Comment:
this is android-specific endpoint. Wouldn't be more appropriate to implement it via executeScript?

Reply:
The execute "endpoints" were meant as temporary stop-gaps. There is no reason to make an Android-specific option map there.